assignments/objects.js

```diff
@@ -19,6 +19,43 @@ const example = {
 
 // Write your intern objects here:
 
+const mitzi = {
+  id: 0,
+  name: "Mitzi",
+  email: "mmelloy0@psu.edu",
+  gender: "F",
+}
+const kennan = {
+  id: 0,
+  name: "kennan",
+  email: "kdiben1@tinypic.com",
+  gender: "M",
+  c3: function(){
```

Comment:
From the question, you arent supposed to add the function directly into the object, like this. 

This works, but it's cheating somewhat.